res/lang/gen_settings/genSettings_en.txt

```diff
@@ -159,43 +159,43 @@ GAMEDAY   = Gameday
 
 
 GAMESPEED      = Game speed
-SET_SPEED_TO_X = Set speed to %SPEED%
+SET_SPEED_TO_X = Set game speed to %SPEED%
 
 MANUAL    = Manual
 HELP      = Help / Pause
 SHOW_HELP = Show help
 
 DO_NOT_SHOW_AGAIN        = Do not show again
 DO_NOT_SHOW_ANY_TIPS     = Do not show any tips
-SHOW_INTRODUCTORY_GUIDES = Show introductory guides
+SHOW_INTRODUCTORY_GUIDES = Show introductory tips
 
 ERROR_NOT_POSSIBLE                  = Not possible!
 ERROR_NOT_ABLE_TO_SELL_LAST_STATION = It is not possible to sell the last station.
 ERROR_NOT_ENOUGH_MONEY              = Not enough money!
-ERROR_NOT_ENOUGH_MONEY_TEXT         = You do not have enough money to finish this action.
+ERROR_NOT_ENOUGH_MONEY_TEXT         = You do not have enough money for this.
 
 NEWGAMESETTINGS_RANDOMIZE          = Randomize
 NEWGAMESETTINGS_RANDOMIZE_DETAIL   = Choose look (color, figure) and names randomly
 NEWGAMESETTINGS_CHANNELNAME_DETAIL = Name of this player's TV channel
 NEWGAMESETTINGS_PLAYERNAME_DETAIL  = Name of this player
 NEWGAMESETTINGS_DIFFICULTY         = Difficulty
-NEWGAMESETTINGS_DIFFICULTY_DETAIL  = The difficulty |b|for|/b| this player
+NEWGAMESETTINGS_DIFFICULTY_DETAIL  = The difficulty for this player
 NEWGAMESETTINGS_START_YEAR_DETAIL  = The year in which the game will start. Affects available programmes!
 NEWGAMESETTINGS_GAME_SEED          = Game number / seed
 NEWGAMESETTINGS_GAME_SEED_DETAIL   = Same game numbers lead to similar starting conditions (offers at vendors).
 
 MESSAGE_CLOSES_AT_TIME = Message closes at %TIME%
-MESSAGE_CLOSES_AT_DAY  = Message closes at %TIME%
+MESSAGE_CLOSES_AT_DAY  = Message closes on %TIME%
 
 
-HINT_PRESSING_ALT_WILL_SHOW_DETAILS    = Pressing "ALT"-key will show details.
-HINT_PRESSING_ALT_WILL_FIX_TOOLTIP     = Pressing "ALT"-key will lock tooltip.
-HINT_PRESSING_ALT_WILL_RELEASE_TOOLTIP = Pressing "ALT"-key will release tooltip.
+HINT_PRESSING_ALT_WILL_SHOW_DETAILS    = Pressing ALT will show details.
+HINT_PRESSING_ALT_WILL_FIX_TOOLTIP     = Pressing ALT will lock tooltip.
+HINT_PRESSING_ALT_WILL_RELEASE_TOOLTIP = Pressing ALT will release tooltip.
 HINT_PROGRAMMEPLANER_SHORTCUTS         = ALT: switch datasheet | CTRL+Click: copy programme | SHIFT+Click: next episode
 HINT                                   = Hint
 
 STARTHINT_TITLE1 = Programme plan
-STARTHINT_TEXT1  = The CTRL-key allows you to place a programme multiple times on your plan, whereas the SHIFT-key tries to place the next episode of a series.
+STARTHINT_TEXT1  = CTRL allows you to place a programme multiple times on your plan, whereas SHIFT places the next episode of a series.
```

Comment:
Removing the "key"-things can lead to inconsistency (style wise) when a description eg. wants to tell you something about the "A"-Key.
Yes, grammar wise it is still OK but might might be less "visible" (dunno,...) when reading it.

Reply:
I was trying to give it a more natural feel, putting "-key" is just awkward and CTRL/SHIFT have obvious meaning.